src/friendly-iframe-embed.js

```diff
@@ -444,6 +444,16 @@ export class FriendlyIframeEmbed {
       this.visibilityObservable_.fire(this.visible_);
     }
   }
+
+  setCssAnimPlayingState(running) {
+    if (platformFor(this.win).isIos()) {
+      this.win.document.documentElement.classList.toggle(
```

Comment:
> a. Sounds to me like we could just change the boilerplate?

It's certainly possible. But (i) we need to fully define it; (ii) we have to do it sooner since this affects other creative authors; (iii) this depends on other issues, e.g. element loading.

> b. The main problem here would be how long the visibility event will take to trigger.

Not just that. Basically, we have a time between creative CSS has been applied and AMP Runtime has been downloaded + installed. While we expect AMP Runtime to be available very quickly most of the time, this is still a major risk condition, which in 99%ile will be seconds.

> Finally, we still need some research on pausing and iOS. My hope that a non-scrollable iframe embed will be ok to use animation-play-state: paused.

Can we try to implement pause/resume via `animation-name: none` and negative delay on resume?